docs/README.md

```diff
@@ -0,0 +1,14 @@
+# Build instructions
+
+To build the docs you must install the following requirements:
+
+- [Hugo 0.147.1](https://github.com/gohugoio/hugo)
```

Comment:
```suggestion
- [Hugo 0.147+](https://github.com/gohugoio/hugo)
```

I don't think we care about a particular release, since I've yet to see breaking changes come through hugo. This change feels like it gives some sense but doesn't tie the docs to a version that might get outdated